tests/shell/tests/01-run.py

```diff
@@ -136,14 +136,17 @@ def check_and_get_bufsize(child):
     return bufsize
 
 
+# there's an issue with some boards' serial that causes lost characters.
+LINE_EXCEEDED_BLACKLIST = {
+    'nrf52dk',
+    'z1',
+}
+
+
 def check_line_exceeded(child, longline):
 
-    if BOARD == 'nrf52dk':
-        # There is an issue with nrf52dk when the Virtual COM port is connected
-        # and sending more than 64 bytes over UART. If no terminal is connected
-        # to the Virtual COM and interfacing directly to the nrf52832 UART pins
-        # the issue is not present. See issue #10639 on GitHub.
```

Comment:
Care to re-add these lines to the nrf52dk entry in the blacklist?

Reply:
done